hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
If you want you can add the copyright part.
In the `operator-sdk init` subcommand there's a `--owner flag`.

See https://github.com/3scale/3scale-operator/blob/master/hack/boilerplate.go.txt for an example where it was created with `--owner "Red Hat"`

Take into account this content is used for the other generated files so changing this manually would not update the other ones.

Reply:
Changed :+1: